src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs

```diff
@@ -347,18 +347,14 @@ private void TryOpenImage()
 			{
 				try
 				{
-					if (imageSource is WriteableBitmap wb)
-					{
-						SetFromWriteableBitmap(wb);
-					}
 					// We want to reset the image when there is no source provided.
-					else if (!imageSource?.HasSource() ?? true)
+					if (!imageSource?.HasSource() ?? true)
 					{
 						ResetSource();
 					}
-					else if (imageSource.ImageData != null)
+					else if (imageSource.TryOpenSync(out var bitmap))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['imageSource' is null on at least one execution path.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5633443)